software/firmware/calibrate.py

```diff
@@ -1,9 +1,10 @@
-from machine import Pin, ADC, PWM
-from time import sleep
+from europi import AnalogueInput, MAX_UINT16
+from machine import Pin, PWM, freq
 
-machine.freq(250000000)
+# Overclock for faster calibration
+freq(250000000)
```

Comment:
lots of 0s here. Underscores would make it more readable.  Does micropython allow that?

Reply:
Good call! I've updated other instances too.